portfolio/src/main/java/com/google/sps/servlets/MyServlet.java

```diff
@@ -0,0 +1,49 @@
+package com.google.sps.servlets;
+
+// import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/information")
+public final class MyServlet extends HttpServlet {
+
+  String[] info = {"Hi", "This is about", "My information"};
+
+  String[] list;
+
+  public MyServlet(String[] list){
+      this.list = list;
+  }
+
```

Comment:
Store your messages as `ArrayList<String>`, you can then use Gson to convert directly from an `ArrayList` to a JSON object that you can return.